rhc-ose-ansible/roles/secure-registry/tasks/main.yaml

```diff
@@ -0,0 +1,164 @@
+---
+
+  - name: "Check that Openshift Docker Registry exists"
+    command: "{{ openshift.common.client_binary }} get deploymentConfig {{ registry_dc }} -n {{ openshift_registry_project }}"
+    register: registry_exists
+    delegate_to: "{{ groups['masters'][0] }}"
```

Comment:
It would be neat if this was a parameter such as `secure_registry_master_host` in order to keep the group names outside the role. Could check for defined prior to doing any work.